using_images/xpaas_images/eap.adoc

```diff
@@ -1,119 +1,110 @@
-= Red Hat JBoss Enterprise Application Platform (JBoss EAP) xPaaS Image
+= Red Hat JBoss Enterprise Application Platform (JBoss EAP) xPaaS Images
 {product-author}
 {product-version}
 :data-uri:
 :icons:
-:experimental:
 :toc: macro
 :toc-title:
+:description: Set up and use xPaaS JBoss EAP 6.4 and 7 Beta images with OpenShift
 
 toc::[]
 
 == Overview
 
-Red Hat JBoss Enterprise Application Platform (JBoss EAP) is available as a containerized xPaaS image that is designed for use with OpenShift. Developers can quickly build, scale, and test applications deployed across hybrid environments.
+Red Hat offers a containerized xPaaS image for the Red Hat JBoss Enterprise Application Platform (JBoss EAP) that is designed for use with OpenShift. Using this image, developers can quickly and easily build, scale, and test applications deployed across hybrid environments.
 
-[IMPORTANT]
-There are significant differences in supported configurations and functionality
-in the JBoss EAP xPaaS image compared to the regular release of JBoss EAP.
-
-This topic details the differences between the JBoss EAP xPaaS image and the
-regular release of JBoss EAP, and provides instructions specific to running and
-configuring the JBoss EAP xPaaS image. Documentation for other JBoss EAP
-functionality not specific to the JBoss EAP xPaaS image can be found in the
-https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/[JBoss
-EAP documentation on the Red Hat Customer Portal].
-
-`_EAP_HOME_` in this documentation, as in the
-https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/[JBoss
-EAP documentation], is used refer to the JBoss EAP installation directory. The
-location of `_EAP_HOME_` inside a JBoss EAP xPaaS image is *_/opt/eap/_*, which
-the `*JBOSS_HOME*` environment variable is set to by default.
+== Comparison Between Product and Image
 
-== Comparing the JBoss EAP xPaaS Image to the Regular Release of JBoss EAP
+The xPaas JBoss EAP images differ from the JBoss EAP product in several ways:
 
-=== Functionality Differences for OpenShift JBoss EAP xPaaS Images
+. The image does not include the JBoss EAP Management Console used to manage xPaaS JBoss EAP images.
+. The JBoss EAP Management CLI is included in the xPaaS JBoss EAP image, but can only access the Management CLI of a container from within the pod.
+. Domain mode is not supported in the xPaaS JBoss EAP image. Instead, OpenShift manages the creation and distribution of applications in the containers.
+. The image’s default root page is disable. Deploy your own application to the root context as ROOT.war.
```

Comment:
Is "ROOT.war" a file name? If so, it should have the correct AsciiDoc markup: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference